packages/suite/src/views/onboarding/steps/Final/index.tsx

```diff
@@ -35,6 +46,10 @@ const StyledButton = styled(Button)`
     }
 `;
 
+const RenameButton = styled(Button)`
+    height: ${({ variant }) => `${INPUT_HEIGHTS[variant as InputVariant]}px`};
```

Comment:
You used a `Button` prop here, which is `undefined`. What I meant is this:

```suggestion
    height: ${INPUT_HEIGHTS.large}px;
```

It is not dynamic, but at least the value is set in one place.